### DISCUSSION TITLE
Discord OAuth2 404

### DISCUSSION THREAD

**Discussion by Subie1** [pocketbase/pocketbase#2490] [Q&A] [answered]
Having an issue whenever I try to run OAuth2 with Discord once the user presses the Authorize button from Discord's OAuth2 page it redirects them to a 404 page from PocketBase

Code:

```js
await pb.collection("users").authWithOAuth2({ provider: "discord" })
```

The error on the page the user is redirected to:

```json
{"code":404,"message":"Not Found.","data":{}}
```

The error on the PocketBase dashboard logs:
```json
{
  "errorDetails": "<nil>",
  "errorMessage": "Not Found"
}
```

Currently using PocketBase version 0.15.3 and ReactJS

I have added the redirect url `http://127.0.0.1:8090/api/oauth2-redirect` to discord

**Comment by ganigeorgiev** [comment]
I don't know how to reproduce it. Could you provide a more complete code sample?

**Reply by Subie1** [reply, marked as answer]
There is no complete code example unfortunately it's just that one liner

my entire code is basically:

```js
const pb = new PocketBase("http://127.0.0.1:8090");

await pb.collection("users").authWithOAuth2({ provider: "discord" });
```
